doc/gui/index.rst

```diff
@@ -1,16 +1,37 @@
-=====
- GUI
-=====
+===
+GUI
+===
 
 The brand-new lightweight web-based hnn GUI that works with hnn-core.
 
+.. raw:: html
 
-Launch GUI in a Jupyter notebook
-================================
+    <div class="sphx-glr-thumbnails">
+        <div class="sphx-glr-thumbcontainer"
+            tooltip="Basic GUI usage">
 
-See :doc:`basic_gui_usage`
+.. only:: html
 
-Event Related Potentials (ERPs)
-===============================
+  .. image:: https://user-images.githubusercontent.com/11160442/197026635-39b6564a-f529-4caf-870e-af9ca4d2fb75.png
+     :alt: 01. Basic GUI usage
 
-See :doc:`tutorial_erp`
+  :doc:`basic_gui_usage`
+
+.. raw:: html
+
+      <div class="sphx-glr-thumbnail-title">01. Basic GUI usage</div>
+    </div>
+    <div class="sphx-glr-thumbcontainer" tooltip="Basic GUI usage">
```

Comment:
Something weird is going on here but I don't know enough about rST + Sphinx Gallery to troubleshoot: [the "Basic GUI usage" tooltip is being applied to both thumbnails](https://output.circle-artifacts.com/output/job/4b3e839d-5c0d-4a39-9162-0a8814bcafb9/artifacts/0/html/gui/index.html).

Reply:
As soon as this is fixed I'll merge @jasmainak!